Source/main.cpp

```diff
@@ -15,23 +15,23 @@
 
 #include <AMReX_Print.H>
 
-int main(int argc, char* argv[])
-{
+int
+main (int argc, char *argv[]) {
     warpx::initialization::initialize_external_libraries(argc, argv);
     {
         WARPX_PROFILE_VAR("main()", pmain);
 
         auto timer = ablastr::utils::timer::Timer{};
         timer.record_start_time();
 
-        auto& warpx = WarpX::GetInstance();
+        auto &warpx = WarpX::GetInstance();
```

Comment:
Having a formatting standard is a good idea - thanks!
However, my own preference is to have a space before and after the ampersand, since to me it is easy to read.
```
auto & warpx = WarpX::GetInstance();
```
In `.clang-format`, this would be
```
ReferenceAlignment: Middle
```

Reply:
Thanks, @dpgrote! This is exactly the kind of feedback process I was hoping for regarding this formatting effort. I am personally fine with your preference, so I implemented the change. Others can chime in as well.